lib/Runtime/Types/JavascriptEnumerator.h

```diff
@@ -40,10 +39,6 @@ namespace Js {
         // seeing this comment please just consolidate the changes.
         virtual Var MoveAndGetNext(PropertyId& propertyId, PropertyAttributes* attributes = nullptr) = 0;
```

Comment:
I still want to change how that work with symbol with my next change.  I will defer the documentation.